simplyblock_core/utils.py

```diff
@@ -1607,6 +1608,14 @@ def set_hugepages_if_needed(node, hugepages_needed, page_size_kb=2048):
     except Exception as e:
         logger.error(f"Node {node}: Error occurred: {e}")
 
+def adjust_hugepages(hugepages):
+    remainder = hugepages % 500
+    hugepages =  hugepages + (500 - remainder)
+
+    str_val = str(hugepages)
+    decimal_val = float(str_val[0] + '.' + str_val[1])
+    add_val = int(decimal_val * 24)
+    return hugepages + add_val
```

Comment:
Its a bit unclear about whats happening here. There seems like a lot of magic numbers used. Why 500? why 24?

maybe you could add comments to function explaining whats happing.

Reply:
What happening here is we calculate the required hugpeage needed for spdk and the same time we need to set this hugepage in k8s pod Request and Limit and they are represented in Gi. 
In a situation spdk request **4966** of hugepage we need to 5Gi as the request and Limit and the representation of 5Gi is **5120**  thus the rounding `5000 + 5.0 * 24`

Reply:
thanks, can you please update the doc string of the func